sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/tests/BlobConfigurationTests.cs

```diff
@@ -65,7 +65,7 @@ public async Task BlobClient_CanConnect_EndPoint()
                 {
                     cb.AddInMemoryCollection(new Dictionary<string, string>()
                     {
-                        {"CustomConnection:endpoint", account.Endpoint },
+                        {"CustomConnection:serviceUri", account.Endpoint },
```

Comment:
should we call this out in changelogs? it's a break between previews isn't it?

Reply:
Yes, good point.